common-rest/src/main/resources/uniprotkb-query.config

```diff
@@ -3,6 +3,7 @@ ec:{query}^1.1
 organism_name:{query}^2.0
 protgene_default:{query}^3.0
 id_default:{query}^10.0
+id_inactive:{query}^10.0
```

Comment:
Hi Edd, I might have missed something, but I did not understand why we are boosting id_inactive. Can you please help me to understand?

Reply:
Good point -- I just treated it as a normal ID in search, but perhaps we shouldn't ... I'm not sure what the value should be. Maybe I'll just leave it unboosted?

The reason the tests work is that it is now a separate field, but you are right -- it is less important and should be boosted less. 

Any suggestions? 2? Or not boosted?

Reply:
I think, not boosted, right? because inactive entries are not important. if we find any, it should be the last to appear, does it make sense to you?